### ISSUE TITLE
"could not serialize access due to read/write dependencies among transactions" error

### ISSUE THREAD

**Issue by nikita-volkov** [nikita-volkov/hasql-transaction#3]
Application fails with the following error when using "hasql-transaction" on concurrent transactions:

```
ResultError (ServerError "40001" "could not serialize access due to read/write dependencies among transactions" (Just "Reason code: Canceled on identification as a pivot, during commit attempt.") (Just "The transaction might succeed if retried."))
```

**Comment by nikita-volkov** [comment]
Fixed in 0.4.5.1.

@begriffs, please beware

**Comment by begriffs** [comment]
Thanks for letting me know, I'll upgrade to this version.
